scripts/core/editor3/components/Editor3.jsx

```diff
@@ -0,0 +1,224 @@
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Editor, EditorState, RichUtils, ContentState} from 'draft-js';
+
+export class Editor3 extends React.Component {
+  constructor(props) {
+    super(props);
+    var initialContentState = ContentState.createFromText(props.value);
+    this.state = {editorState: EditorState.createWithContent(initialContentState)};
+
+    this.readOnly = props.readOnly || false;
+    this.showToolbar = props.showToolbar || false;
+    this.editorFormat = props.editorFormat || [];
+    this.parentOnChange = props.onChange;
+
+    this.focus = () => this.refs.editor.focus();
+    this.onChange = (editorState) => this._onChange(editorState);
+
+    this.handleKeyCommand = (command) => this._handleKeyCommand(command);
+    this.onTab = (e) => this._onTab(e);
```

Comment:
I'm just curious - why are we not defining this methods as public? Why do you use this pattern where you declare a public variable that exposes a private method with no changes?